### DISCUSSION TITLE
Feature Request: Being able to approve/comment/request changes for pull requests without opening the files tab

### DISCUSSION THREAD

**Discussion by ghost** [community/community#35103] [Pull Requests]
Hello I'm working on a project at work that uses GitHub and it uses binary files for 75% of its files, and sometimes pull request are adding/removing/changing hundreds or sometimes thousands of files(this is due to the nature of our project and with it being in the video game industry with a lot of assets) and it takes a while for the `Files Changed` tab to load just so we can just click on the Pull Request review button and move on with our jobs because the button is unclickable while its loading all of the changed files(this might be an oversight from the GitHub team but still its something that does slow down our workflow from an outside source and we are investigating alternative hosting platforms because of this).

I wanted to suggest adding a repository setting for exposing the Pull Request review button in the Conversation tab to quickly be able to respond to pull requests that don't need to check the files changed tab, I'm totally fine with it having like a little message under it saying "This reviewer did not review files changed" or something as a full disclosure type of message because this is meant to help speed up workflows waiting for GitHub to load the changed files.

**Comment by tonglil** [comment]
+1

Similar request in https://github.com/orgs/community/discussions/11227.

**Comment by robmcgee-isw** [comment]
+1
